src/Common/src/System/SR.cs

```diff
@@ -89,3 +97,20 @@ internal static string Format(string resourceFormat, object p1, object p2, objec
         }
     }
 }
+
+namespace System.Resources.Diagnostics
+{
+    [AttributeUsage(AttributeTargets.All)]
+    [ExcludeFromCodeCoverage]
+    internal sealed class ExcludeFromCodeCoverageAttribute : Attribute
```

Comment:
Sorry I missed your other comment about this already existing. In which case how do we prevent them from conflicting?